react-js/react-part-2.md

```diff
@@ -0,0 +1,184 @@
+# React Part 2
+
+## Projected Time
+
+1.5 hours
+- Lesson: 30min
+- Guided Practice: 30min
+- Independent Practice: 30min
+
+### Prerequisites
+
+- [React](react.md)
+- [JSX](react.md)
+- [Props](react.md)
```

Comment:
These are prerequisite lessons, not concepts so please turn into a single prereq